.gitignore

```diff
@@ -16,3 +16,6 @@ _pycache_/
 
 # Python
 *.pyc
+*.orig
+*.pyc.*
```

Comment:
wouldn't this be handled by the *.pyc above?
I made this comment already in PR #46  so it looks like these have to be merged in order.

Reply:
No, becasue the files look like this:
conftest.cpython-36-pytest-5.4.2.pyc.1764

When I try to remove '*.pyc.*', all the files in the pattern above were unstaged in Git.

Reply:
We will be glad that this PR will be merged before #46 😃